packages/core/lib/commands/config.js

```diff
@@ -158,6 +166,11 @@ const parse = function (args) {
 
       break;
     }
+    case "list": {
+      set = false;
```

Comment:
You shouldn't need to set the value for `set` as it is already `false`. Actually I have no idea why this happens on line 120 either.

Reply:
Done.